lib/registry-secrets.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2018 Balena
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
@hedss, is this the boilerplate you mentioned in the [other PR's comment](https://github.com/balena-io/resin-builder/pull/562#discussion_r238280047)?

Reply:
@pdcastro A recent boilerplate thread is here: https://www.flowdock.com/app/rulemotion/r-product/threads/QRTbDzbq6z7C_Xa5DwS7D3Vk8TG

As such, I've adjusted it accordingly in the projects I've been involved in. You can see a copy of the current one here: https://github.com/balena-io/balena-certificate-dns-service/blob/master/src/app.ts#L1